src/commands/org/create/sandbox.ts

```diff
@@ -136,10 +136,17 @@ export default class CreateSandbox extends SandboxCommandBase<SandboxProcessObje
 
   private async createSandboxRequest(): Promise<SandboxRequest> {
     // reuse the existing sandbox request generator, with this command's flags as the varargs
-    const { sandboxReq } = await createSandboxRequest(false, this.flags['definition-file'], undefined, {
+    const requestOptions = {
       ...(this.flags.name ? { SandboxName: this.flags.name } : {}),
+      ...(this.flags.clone ? { SourceSandboxName: this.flags.clone } : {}),
       ...(!this.flags.clone && this.flags['license-type'] ? { LicenseType: this.flags['license-type'] } : {}),
```

Comment:
the error the user is reporting look like oclif errors because of the oclif `exclusive` aren't they?

Reply:
Yes they are exclusive, but I tracked down the error to createSandboxRequest, in the [else block here](https://github.com/salesforcecli/plugin-org/blob/e5b941ade71ab2888b75e3fdff0ffdbc856bb0a1/src/shared/sandboxRequest.ts#L64). `isClone` is false causing the license-type required message in the else block to emit.

Not sure if that code needs to be reworked, but I can see the necessity of the check since this code is once removed from the command.

Reply:
Won't you need to fix the `exclusive` property on the flags?  Are those errors in the github issue coming from the oclif flag parsing?

Reply:
@mshanemc No, the flags are exclusive and not being used with each other. The error is coming from https://github.com/salesforcecli/plugin-org/blob/main/src/shared/sandboxRequest.ts#L74